source/GooglePlayServices-Firebase.Solution.proj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.Build.Traversal/2.1.1">
+    <ItemGroup>
+        <SolutionConfiguration Include="Debug"/>
+    </ItemGroup>  
+    <ItemGroup>
+        <!-- Build all projects recursively under the "src" folder -->
+        <ProjectReference Include=".\**\*.*proj" />
```

Comment:
What does this do? Is it not already added to the generated solution?

Reply:
I played with solution replacement (Microsoft.Build.Traversal)